src/Geometry/GMUGeoJSONParser.m

```diff
@@ -106,7 +106,7 @@ - (instancetype)initWithStream:(NSInputStream *)stream {
 
 - (instancetype)initWithURL:(NSURL *)url {
   if (self = [super init]) {
-    _stream = [[NSInputStream alloc] initWithURL:url];
+    _data = [[NSData alloc] initWithContentsOfURL:url];
```

Comment:
Yeah, after using this, we are going to have to do something smarter before launching 2.0.